src/rules/prefer-expect-assertions.ts

```diff
@@ -84,6 +83,7 @@ export default createRule<[RuleOptions], MessageIds>({
       suggestRemovingExtraArguments: 'Remove extra arguments',
     },
     type: 'suggestion',
+    hasSuggestions: true,
```

Comment:
Can you send a PR with this one to main?

Reply:
It's already in `main` (see #898 & #899), but `main` needs to be merged into `next` first

Reply:
ah, cool. I can merge it in 🙂

Reply:
done